src/buildevents.ts

```diff
@@ -28,8 +28,15 @@ export async function install(apikey: string, dataset: string): Promise<void> {
 }
 
 export function addFields(keyValueMap: object): void {
-  fs.writeFileSync('../buildevents.txt', logfmt.stringify(keyValueMap))
-  util.setEnv('BUILDEVENT_FILE', '../buildevents.txt')
+  const envPath = util.getEnv('BUILDEVENT_FILE') || '../buildevents.txt'
```

Comment:
Oh, this is a good idea! I assume you've set this because you write to the buildevents file yourself as well?